sdk/batch/Microsoft.Azure.Batch/tests/IntegrationTests/IntegrationTestUtilities/FileUploadUtilities.cs

```diff
@@ -252,7 +252,7 @@ private static string ConstructBlobSource(string container, string blob)
         /// <param name="permissions">permission on the name</param>
         /// <param name="blobUri">blob URI</param>
         /// <returns>the SAS for the container, in full URI format.</returns>
-        private static string CreateContainerWithPolicySASIfNotExist(string account, string key, Uri blobUri, string container, string policy, DateTime start, DateTime end, SharedAccessBlobPermissions permissions)
+        private static async Task<string> CreateContainerWithPolicySASIfNotExist(string account, string key, Uri blobUri, string container, string policy, DateTime start, DateTime end, SharedAccessBlobPermissions permissions)
```

Comment:
Minor: If this is an async function now, can you add an async suffix here.

Reply:
Ok, will do.